app/assets/javascripts/client/home/home_anon.html.erb

```diff
@@ -1,103 +1,66 @@
-<section class='hero'>
-  <div class='row'>
-    <div class='col-sm-12 text-center'>
-      <h1>PDredesign</h1>
-    </div>
-  </div>
-  <div class='row'>
-    <div class='col-sm-12 text-center'>
-      <p>A digital toolkit and community of practice</br>
-        dedicated to improving professional</br>
-        development for teachers.</p>
-    </div>
+<div class="hero-header">
+  <div class="hero-banner">
+    <h1>PDredesign</h1>
+    <p class="text-center">A digital toolkit and community of practice dedicated to improving
+      <u>professional development</u> for teachers.</p>
```

Comment:
Not a big deal at all, but I think non-semantic/style tags should play no role in HTML. I'd have used `<span class="underlined">` instead.